### PR TITLE
WIP Doctopus navigation injection

### PR DESCRIPTION
**Pull request by jarofghosts**

### CODE REVIEW

**Comment by Gastove** [comment]
DIG IT

**Comment by gmcquillan** [comment]
Looks 200%

File: src/doctopus/web.clj [hunk 1]
```diff
@@ -34,16 +35,42 @@
       response
       (four-oh-four (:uri request)))))
 
-(defn serve-index
-  "Just returns our index.html, with the content-type set correctly"
-  [_]
-  (-> (slurp (io/resource "index.html"))
+(defn serve-html
```
Comment (Gastove) [inline src/doctopus/web.clj:39]: I recognize this :dancer:

File: src/doctopus/web.clj [hunk 2]
```diff
@@ -34,16 +35,42 @@
...
+  "Just returns html, with the content-type set correctly"
+  [template]
```
Comment (jarofghosts) [inline src/doctopus/web.clj:41]: this should probably be `html` vs. `template`

File: src/doctopus/web.clj [hunk 3]
```diff
@@ -34,16 +35,42 @@
...
+
+(defn serve-add-head-form
+  [_]
+  (serve-html (templates/add-head)))
+
+(defn serve-add-tentacle-form
+  [_]
+  (serve-html (templates/add-tentacle)))
+
+(defn add-head
```
Comment (Gastove) [inline src/doctopus/web.clj:58]: Probably eventually gonna want some of this to live in a different NS, just to keep stuff tidy?

Comment (jarofghosts) [inline src/doctopus/web.clj:58]: definitely. i plan on refactoring it soon

**Pull request merged**